test/index.test.js

```diff
@@ -154,3 +155,17 @@ test('Create the error without the new keyword', t => {
   t.equal(err.statusCode, 500)
   t.ok(err.stack)
 })
+
+test('isFastifyError', t => {
+  t.plan(8)
+
+  const NewError = createError('CODE', 'Not available')
+  t.equal(isFastifyError(NewError()), true)
```

Comment:
I wonder if we should consider the `createError` instances as "application error" instead of Fastify Error